basic/basic.go

```diff
@@ -35,11 +43,19 @@ type Exporter struct {
 }
 
 // New creates a new instance of a Exporter.
-func New(config *config.Config, sessions *sessions.Sessions) *Exporter {
+// enableOverlapping is using for backward compatibility.
+// See: https://jira.percona.com/browse/PMM-1901.
+func New(config *config.Config, sessions *sessions.Sessions, enableMetrics OverlappingMetrics) *Exporter {
```

Comment:
s/enableMetrics/enableOverlapping/
That's how the comment above calls this parameter.